good-enough-practices-for-scientific-computing.tex

```diff
@@ -1169,7 +1169,23 @@ \section{What We Left Out}\label{sec:omitted}
 
 \section{Conclusion}\label{sec:conclusion}
 
-\fixme{GW}{write a conclusion}
+We have outlined a series of practices for scientific computing based
+our collective experience, and the experience of the thousands of
+researchers we have met through Software Carpentry, Data Carpentry,
+and similar organizations.  These practices are pragmatic, accessible
+to people who consider themselves computing neophytes and can be
+applied by both individuals and groups.
+
+Most importantly, we believe that practices like these are
```

Comment:
Should we first mention the benefits to the individual researcher (e.g. more efficient research) before the more altruistic reproducibility goals?